tool/site/client-react/src/fhir/helpers.ts

```diff
@@ -65,11 +64,6 @@ export function getRepeatsIssuedAndAllowed(medicationRequest: MedicationRequest)
   const numberOfRepeatPrescriptionsIssued = ukCoreRepeatsIssuedExtension
     ? ukCoreRepeatsIssuedExtension.valueUnsignedInt
     : 1
-
-  const ukCoreRepeatsAllowedExtension = getUkCoreNumberOfRepeatsAllowedExtension(medicationRequest.extension)
-  const numberOfRepeatPrescriptionsAllowed = ukCoreRepeatsAllowedExtension
-    ? ukCoreRepeatsAllowedExtension.valueUnsignedInt
-    : (medicationRequest.dispenseRequest.numberOfRepeatsAllowed || 0) + 1
-
+  const numberOfRepeatPrescriptionsAllowed = medicationRequest.dispenseRequest?.numberOfRepeatsAllowed || 1
```

Comment:
Think this should be `(medicationRequest.dispenseRequest.numberOfRepeatsAllowed || 0) + 1`